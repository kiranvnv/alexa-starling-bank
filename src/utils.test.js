const Utils = require('./utils');

describe('buildSpeechResponse', () => {
  it('returns a valid Alexa response', () => {
    expect(Utils.buildSpeechResponse('Hello!')).toMatchSnapshot();
  });

  describe('setting shouldEndSession', () => {
    it('returns a valid Alexa response', () => {
      expect(Utils.buildSpeechResponse('Hello!', false)).toMatchSnapshot();
    });
  });
});

