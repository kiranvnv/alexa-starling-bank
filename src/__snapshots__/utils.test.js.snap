// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`buildSpeechResponse returns a valid Alexa response 1`] = `
Object {
  "response": Object {
    "outputSpeech": Object {
      "text": "Hello!",
      "type": "PlainText",
    },
    "shouldEndSession": true,
  },
  "version": "1.0",
}
`;

exports[`buildSpeechResponse setting shouldEndSession returns a valid Alexa response 1`] = `
Object {
  "response": Object {
    "outputSpeech": Object {
      "text": "Hello!",
      "type": "PlainText",
    },
    "shouldEndSession": false,
  },
  "version": "1.0",
}
`;
